index.html

```diff
@@ -2228,7 +2228,10 @@ <h2 id="docconformance">
                 <code>role=<a href="#index-aria-cell">cell</a></code> if a
                 descendant of a `table` element.
               </p>
-              <!--<p><code>role=<a href="#index-aria-cell">gridcell</a></code> if a descendant of a <code>table</code> elementwith <code>role=grid</code></p>-->
+              <p>
+                <code>role=<a href="#index-aria-gridcell">gridcell</a></code> if a
```

Comment:
we want to verify this actually is true in browsers.  it _should_ be, but if not, then we'd want to note a special case for it and file bugs where this doesn't hold true.